frontend/src/app/login-form/login-form.component.html

```diff
@@ -0,0 +1,42 @@
+<div class="login-container">
+  <mat-card class="login-card">
+    <mat-card-header class="login-header">
+      <div mat-card-avatar class="login-card-icon"></div>
+      <mat-card-title>Sign In</mat-card-title>
+    </mat-card-header>
+    <mat-card-content class="login-card-content">
+      <mat-form-field appearance="fill">
+        <mat-label>Username</mat-label>
+        <input matInput type="text" value="ala" #username />
+      </mat-form-field>
+      <mat-form-field appearance="fill">
+        <mat-label>Password</mat-label>
+        <input
+          matInput
+          [type]="hide ? 'password' : 'text'"
+          value="makota"
```

Comment:
I guess the inputs shouldn't have a default value.

Reply:
I did on purpose due to testing

Reply:
As per continuation of our Discord chat: just make sure you'll remove it before merge :)

Reply:
ok, thanks for pointing this out :)